docs/documentation/pages/internal/DEVELOPMENT.md

```diff
@@ -409,6 +409,35 @@ spec:
         image: {{ include "helm_lib_module_common_image" (list . "kubeRbacProxy") }}
 ```
 
+### Service accounts
+
+Every service account used in deckhouse modules should have appropriate flow schema to prevent API overloading.
+
+In deckhouse we have 4 predefined PriorityLevelConfigurations for use in flow schemas:
```

Comment:
```suggestion
Every service account used in Deckhouse modules should have an appropriate flow schema to prevent API overloading.

There are 4 predefined `PriorityLevelConfigurations` in Deckhouse for use in flow schemas:
```